modules/mono/glue/GodotSharp/GodotSharp/Core/StringExtensions.cs

```diff
@@ -426,7 +426,9 @@ private static string CamelcaseToUnderscore(this string instance, bool lowerCase
         /// <returns>An integer that indicates the lexical relationship between the two comparands.</returns>
         public static int CasecmpTo(this string instance, string to)
         {
-            return instance.CompareTo(to, caseSensitive: true);
+#pragma warning disable CA1309
```

Comment:
nit: Add diagnostic title.

```suggestion
#pragma warning disable CA1309 // Use ordinal string comparison
```